utils/appliance/__init__.py

```diff
@@ -2378,20 +2419,65 @@ def pop(self):
 stack = ApplianceStack()
 
 
+def parallel_appliances_loader(appliance_list, global_kwargs):
```

Comment:
Consider calling 'load_parallel_appliances' for consistency with method name below (+_from_confg)